src/main/resources/view/HelpWindow.fxml

```diff
@@ -82,6 +82,7 @@
                             <UG action="Find" format="find KEYWORD [MORE_KEYWORDS]…" />
                             <UG action="List" format="list" />
                             <UG action="View" format="view IDNUMBER" />
+                            <UG action="Schedule" format="schedule DATE" />
```

Comment:
date is optional, place the [] around it

Reply:
![image](https://user-images.githubusercontent.com/77286511/113545901-9e5c5000-961d-11eb-836a-0e5ef1211fa0.png)
Then this is wrong coz non optional parameters also have []

Reply:
![image](https://user-images.githubusercontent.com/77286511/113546049-dfecfb00-961d-11eb-8e2e-6e0a82e05405.png)
![image](https://user-images.githubusercontent.com/77286511/113546172-188cd480-961e-11eb-9656-a8fe5eb92427.png)


Everything should follow this right? Then the most of the help command formats are wrong.

Reply:
I think the better way to phrase it is that everything should be in sync and mirror each other when accurate. Two wrongs doesn't make one right. You might want to just update all since these are all documentation issues.

Reply:
So follow the format in UG?

Reply:
I don't think there's a clear cut answer to your query, I'm not sure if the UG is fully accurate as well, the only correct format is what the application currently accepts.